qa/workunits/cephadm/test_repos.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
 
 SCRIPT_NAME=$(basename ${BASH_SOURCE[0]})
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]})
```

Comment:
```suggestion
SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
```

Reply:
This change will make the path relative instead of absolute .. but it's probably ok as long as the test script doesn't `cd` into another path ...